fakturogen-core/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
```

Comment:
Current parent is for main-root module. This parent should be here:
<parent>
        <artifactId>fakturogen-parent</artifactId>
        <groupId>pl.fakturogen</groupId>
        <version>1.0-SNAPSHOT</version>
    </parent>

Reply:
Agree.